src/TEC/Import/Post_Handler.php

```diff
@@ -935,6 +935,54 @@ private function update_linked_post_meta( string $linked_post_type, string $meta
 		}
 	}
 
+	/**
+	 * Relinks posts to a series during import.
+	 *
+	 * @since 1.2.0
+	 *
+	 * @param mixed  $value              The meta value to be imported
+	 * @param int    $post_id            The ID of the post being imported
+	 * @param string $key                The meta key being processed
+	 * @param mixed  $original_value     The original meta value before processing
+	 * @param array  $existing_meta_keys Existing meta keys for the post
+	 * @param int    $import_id          The ID of the current import
+	 *
+	 * @return string Empty string to prevent meta saving, or original value if conditions not met
+	 */
+	public function relink_posts_to_series( $value, $post_id, $key, $original_value, $existing_meta_keys, $import_id ) {
+		// Bail if not series.
+		if ( get_post_type( $post_id ) !== 'tribe_event_series' ) {
+			return $value;
+		}
+
+		// Bail if not the correct meta key.
+		if ( $key !== 'posts_in_series' ) {
+			return $value;
+		}
+
+		// Explode the value which are the post IDs.
+		$post_ids = explode( ',', $value );
+		// Get the Series post object
+		$series = get_post( $post_id );
+
+		$this->add_to_log( "Fetching new post IDs for the Series..." );
+
+		// Grab the new post IDs based on the hash.
+		$new_post_ids = array_filter( array_map( function ( $post_id ) {
+			$hash = $this->hashit( $post_id );
+
+			return $this->get_post_id_from_meta( '_tribe_events_export_hash', $hash );
+		}, $post_ids ) );
+
+		$this->add_to_log( "Relinking posts to the Series..." );
+		// Get the relationship class and relink the posts.
+		$relationship = new \TEC\Events_Pro\Custom_Tables\V1\Series\Relationship();
+		$relationship->with_series( $series, $new_post_ids );
```

Comment:
Here you will need to invalidate cache for the Events you have updated and for the Series post.

Reply:
How would I do that?
Would it be something like
```Cache_Listener::instance()->save_post( $post_id, get_post( $post_id ) );```

Reply:
You should use `clean_pos_cache( $post_id )` for each post.
Under the hood that call will also take care to clear the request (memoization) and object cache handled / managed by the TEC cache.

Reply:
Done.